platformio.ini

```diff
@@ -4,6 +4,7 @@
 [platformio]
 default_envs = esp8266
 ; default_envs = esp32
+; default_envs = esp32_d1
```

Comment:
@MichaelDvP I'd rather keep the platformio.ini clean and generic. You can move your specific settings to `pio_local.ini` which will override anything in `platformio.ini`